Assets/HoloToolkit/ARCA/Scripts/SpatialSync/AnchorLocated.cs

```diff
@@ -1,43 +1,67 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
-
+// Licensed under the MIT License. See LICENSE in the project root for license information.
 using UnityEngine;
 using UnityEngine.XR.iOS;
 
 namespace HoloToolkit.ARCapture
 {
+    /// <summary>
+    /// Detects when an anchor has been located
+    /// </summary>
     public class AnchorLocated : MonoBehaviour
     {
-        [Tooltip("The 3D marker generator")]
-        public ARCAMarkerGenerator3D MarkerGenerator;
+        /// <summary>
+        /// Delegate for when an achor is located
+        /// </summary>
         public delegate void AnchorLocatedEvent();
+
+        /// <summary>
+        /// The 3D marker generator
+        /// </summary>
+        [SerializeField] [Tooltip("The 3D marker generator")]
+        private ARCAMarkerGenerator3D markerGenerator;
+
+        /// <summary>
+        /// Callback when an anchor is located by the HoloLens
+        /// </summary>
         public AnchorLocatedEvent OnAnchorLocated;
 
-        bool transitioned = false;
+        /// <summary>
+        ///
+        /// </summary>
+        private bool transitioned;
 
-        void Start()
+        /// <summary>
+        /// The 3D marker generator
+        /// </summary>
+        public ARCAMarkerGenerator3D MarkerGenerator
         {
-            if(MarkerGenerator == null)
-            {
-                MarkerGenerator = FindObjectOfType<ARCAMarkerGenerator3D>();
-            }
+            get { return markerGenerator; }
+            set { markerGenerator = value; }
+        }
+
+        private void Start()
+        {
+            if (MarkerGenerator == null) MarkerGenerator = FindObjectOfType<ARCAMarkerGenerator3D>();
             UnityARSessionNativeInterface.ARFrameUpdatedEvent += FrameUpdated;
         }
 
-        void OnDestroy()
+        private void OnDestroy()
         {
             UnityARSessionNativeInterface.ARFrameUpdatedEvent -= FrameUpdated;
         }
 
-        void FrameUpdated(UnityARCamera camera)
+        /// <summary>
+        /// Called by the API. It checks whether an anchor has been located and signals
+        /// the marker generator so that it can create and show an AR marker
+        /// </summary>
+        /// <param name="cam"></param>
+        private void FrameUpdated( UnityARCamera cam )
         {
-            if(camera.pointCloudData.Length > 4)
+            if (cam.pointCloudData.Length > 4)
```

Comment:
Just a suggestion. I'm more in favor of  `camera` vs `cam`.  Generally abbreviations should be avoided.

Reply:
I totally agree about avoiding abbreviations. 
However, even if deprecated, ```camera``` is still a member of ```MonoBehaviour``` and I think we can skip that rule in this case to avoid ambiguities

Reply:
Agreed. `arCamera` should be good here then.